packages/react/src/components/TextArea/TextArea.tsx

```diff
@@ -129,10 +241,10 @@ const TextArea = React.forwardRef(function TextArea(
     <textarea
       {...other}
       {...textareaProps}
-      placeholder={placeholder || null}
+      placeholder={placeholder}
       className={textareaClasses}
-      aria-invalid={invalid || null}
-      aria-describedby={invalid ? errorId : null}
+      aria-invalid={invalid}
+      aria-describedby={invalid ? errorId : undefined}
```

Comment:
@alisonjoseph do we want this to be undefined or remain `: null` curious on your thoughts!

Reply:
The reason for these changes is to match the expected prop types on the `<textarea/>` element.  Otherwise, typescript shows the following error. 
```
    Types of property ''aria-describedby'' are incompatible.
      Type 'string | null' is not assignable to type 'string | undefined'.
        Type 'null' is not assignable to type 'string | undefined'.ts(2322)
```

Reply:
Ohh gotcha, thank you for clarifying :)